tools/cldr-code/src/main/java/org/unicode/cldr/util/UnitConverter.java

```diff
@@ -1851,28 +1833,41 @@ public enum UnitSystem { // TODO convert getSystems and SupplementalDataInfo to
         si_acceptable,
```

Comment:
Is the order significant -- possibly affecting order of rows/pages in ST for example? If so, comment would be helpful, otherwise alphabetical order might be appropriate

Reply:
It would affect the order of rows within the pages, grouping the quantities better.